core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -118,19 +129,68 @@ private[op] class OpMultiClassificationEvaluator
         thresholds = $(thresholds)
       )
 
+      val metricsTopK = calculateTopKMetrics(
+        data = dataUse.select(col(probabilityColName), col(labelColName).cast(DoubleType)).rdd.map{
```

Comment:
Let's do the RDD conversion once since we're sending the same one to the threshold metrics calculation